db/sqlinterfaces.c

```diff
@@ -7397,6 +7393,17 @@ CDB2QUERY *read_newsql_query(struct sqlclntstate *clnt, SBUF2 *sb)
     return query;
 }
 
+int gbl_debug_high_availability_flag = 0;
+static inline void set_high_availability(struct sqlclntstate *clnt, int val)
+{
+    clnt->high_availability = val;
```

Comment:
Maybe rename this to make sure there are no other direct accesses so that you can verify that you caught all of these